tests/app/questionnaire/test_questionnaire_schema.py

```diff
@@ -1,503 +1,233 @@
-                        {
-                            'id': 'group-1',
-                            'blocks': [{'id': 'block-1', 'type': 'Confirmation'}],
-                        }
-                    ],
-                }
-            ]
-        }
-
-        schema = QuestionnaireSchema(survey_json)
-        self.assertTrue(schema.is_confirmation_section(schema.get_section('section-1')))
-        self.assertTrue(schema.is_confirmation_group(schema.get_group('group-1')))
-        self.assertFalse(schema.is_summary_section(schema.get_section('section-1')))
-        self.assertFalse(schema.is_summary_group(schema.get_group('group-1')))
-
-    def test_get_group_for_list_collector_child_block(self):
-        survey_json = {
-            'sections': [
-                {
-                    'id': 'section1',
-                    'groups': [
-                        {
-                            'id': 'group',
-                            'blocks': [
-                                {
-                                    'id': 'list-collector',
-                                    'type': 'ListCollector',
-                                    'for_list': 'list',
+
+
+def test_get_sections(single_question_schema):
+    schema = QuestionnaireSchema(single_question_schema)
+    assert len(schema.get_sections()) == 1
+
+
+def test_get_section(single_question_schema):
+    schema = QuestionnaireSchema(single_question_schema)
+    section = schema.get_section('section1')
+    assert section['title'] == 'Section 1'
+
+
+def test_get_blocks(single_question_schema):
+    schema = QuestionnaireSchema(single_question_schema)
+    assert len(schema.get_blocks()) == 1
+
+
+def test_get_block(single_question_schema):
+    schema = QuestionnaireSchema(single_question_schema)
+    block = schema.get_block('block1')
+
+    assert block['id'] == 'block1'
+
+
+def test_get_groups(single_question_schema):
+    schema = QuestionnaireSchema(single_question_schema)
+    assert len(schema.get_groups()) == 1
+
+
+def test_get_group(single_question_schema):
+    schema = QuestionnaireSchema(single_question_schema)
+    group = schema.get_group('group1')
+
+    assert group['title'] == 'Group 1'
+
+
+def test_get_questions_with_variants(question_variant_schema):
+    schema = QuestionnaireSchema(question_variant_schema)
+    questions = schema.get_questions('question1')
+
+    assert len(questions) == 2
+    assert questions[0]['title'] == 'Question 1, Yes'
+    assert questions[1]['title'] == 'Question 1, No'
+
+
+def test_get_questions(single_question_schema):
+    schema = QuestionnaireSchema(single_question_schema)
+    questions = schema.get_questions('question1')
+
+    assert questions[0]['title'] == 'Question 1'
+
+
+def test_schema_answers(single_question_schema):
+    schema = QuestionnaireSchema(single_question_schema)
+    answers = schema.get_answer_ids()
+    assert len(answers) == 1
+
+
+def test_get_answers_with_variants(question_variant_schema):
```

Comment:
I think it would be clearer if these method names included the method name being tested. For example this could be `test_get_answers_by_answer_id_with_variants`.